Formula/clamav.rb

```diff
@@ -44,7 +45,7 @@ def install
       --with-llvm=no
       --with-libiconv-prefix=#{Formula["libiconv"].opt_prefix}
       --with-iconv=#{Formula["libiconv"].opt_prefix}
-      --with-libjson-static=#{Formula["json-c"].opt_prefix}/lib/libjson-c.a
+      --with-libjson=#{Formula["json-c"].opt_prefix}
```

Comment:
Can I assume this change was done because of a build error in clamav when using the json-c built by this recipe?  Perhaps we can revert this change in the next json-c release (0.15).  I forgot that the cmake tooling for 0.14 doesn't build with `-fPIC`.  It should be fixed in the next release (see https://github.com/json-c/json-c/issues/508)

Reply:
No, it's because it doesn't provide static libraries by default anymore and homebrew prefers dynamic libraries.

Reply:
Perhaps that stance on dynamic libraries could change in the future? Here's a pretty compelling argument why static libraries should be preferred over dynamic https://twitter.com/Cor3ntin/status/1276853471548002304

In this case, while the static json-c 0.14 library won't build in libclamav without the fix I mentioned to add `-fPIC` to CFLAGS, there are stability concerns when including the dynamic json-c library.  json-c shares symbol names with jansson, meaning applications that use json-c via a library like libclamav and also use jansson for their own json needs will see symbol collisions (i.e. random crashes) because the application has to link with both json-c and jansson dynamic libraries.  For this reason, I would recommend including libjson-c.a in the homebrew package when json-c 0.15 comes out.  For 0.14 it doesn't make as much difference [to clamav] because static linking to a .so will fail w/out `-fPIC`.

Reply:
Json-c removed it from their default install, while we could add it back we follow upstream's defaults unless those are known to be insufficient. You should instead be asking json-c to revert their change.

Reply:
It looks like someone has already addressed this a couple months ago for the next release: https://github.com/json-c/json-c/commit/e97fc20bfd99d341174dc61c30ce7f9540786a6f